apache/incubator-zeppelin
# ZEPPELIN-528 REST API: add "progress" to 'get notebook job' API

### What is this PR for?

To add 'progress' to 'get notebook job' REST API. 
All paragraphs which status is 'running' will include 'progress' to their job status.

### What type of PR is it?

Improvement

### Todos

### Is there a relevant Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-528

### How should this be tested?

1. Run a notebook or paragraph which takes some times. Paragraph which could show progress (%) would be more appreciated. Confirm that its status is 'RUNNING'.
2. Call ```http://<zeppelin host>:<zeppelin port>/api/notebook/job/<notebook id>``` to see if it has 'progress' field in JSON.

### Screenshots (if appropriate)

![notebook-2b68v95jm](https://cloud.githubusercontent.com/assets/1317309/11949192/cf176d8e-a8bf-11e5-9bb2-8f4dbd1666c8.png)
![notebook-job-api-result-2b68v95jm](https://cloud.githubusercontent.com/assets/1317309/11949191/cf16f7e6-a8bf-11e5-825b-97c6fa227d05.png)

### Questions:
* Does the licenses files need update? (No)
* Is there breaking changes for older versions? (No)
* Does this needs documentation? (Yes or no. Output of the JSON format could be updated, but it could be treated as not mandatory.)
